libs/application/templates/transport-authority/transfer-of-vehicle-ownership/src/fields/CoOwner/index.tsx

```
@@ -51,70 +51,83 @@ export const CoOwner: FC<React.PropsWithChildren<FieldBaseProps>> = (props) => {
-                </GridColumn>
-              </GridRow>
-            </Box>
-          ),
+          (coOwner: VehiclesCurrentOwnerInfo, index: number) => {
+            const nationalIdField = `${id}[${index}].nationalId`
+            const nameField = `${id}[${index}].name`
+            const emailField = `${id}[${index}].email`
+            const phoneField = `${id}[${index}].phone`
+
+            return (
+              <Box marginTop={3}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)